<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ArgPropertiesBase</class>
 <widget class="QDialog" name="ArgPropertiesBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>526</width>
    <height>490</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Argument Properties</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Name</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="name"/>
     </item>
     <item>
      <widget class="QCheckBox" name="unnamedCb">
       <property name="text">
        <string>Unnamed</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Python type</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="pyType"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Annotations</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QCheckBox" name="allowNoneCb">
        <property name="text">
         <string>AllowNone</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QCheckBox" name="noCopyCb">
        <property name="text">
         <string>NoCopy</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="arrayCb">
        <property name="text">
         <string>Array</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QCheckBox" name="outCb">
        <property name="text">
         <string>Out</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QCheckBox" name="arraySizeCb">
        <property name="text">
         <string>ArraySize</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QCheckBox" name="pyIntCb">
        <property name="text">
         <string>PyInt</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QCheckBox" name="constrainedCb">
        <property name="text">
         <string>Constrained</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <layout class="QFormLayout" name="formLayout_2">
        <property name="fieldGrowthPolicy">
         <enum>QFormLayout::FieldsStayAtSizeHint</enum>
        </property>
        <item row="0" column="1">
         <widget class="QLineEdit" name="ref"/>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="refLab">
          <property name="text">
           <string>Reference</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="4" column="0">
       <widget class="QCheckBox" name="disallowNoneCb">
        <property name="text">
         <string>DisallowNone</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QCheckBox" name="resultSizeCb">
        <property name="text">
         <string>ResultSize</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0" rowspan="3">
       <layout class="QFormLayout" name="formLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>DocType</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="docType"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>DocValue</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="docValue"/>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Encoding</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QComboBox" name="encodingCb"/>
        </item>
       </layout>
      </item>
      <item row="5" column="1">
       <widget class="QCheckBox" name="singleShotCb">
        <property name="text">
         <string>SingleShot</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QCheckBox" name="transferCb">
        <property name="text">
         <string>Transfer</string>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QCheckBox" name="transferBackCb">
        <property name="text">
         <string>TransferBack</string>
        </property>
       </widget>
      </item>
      <item row="8" column="0">
       <widget class="QCheckBox" name="getWrapperCb">
        <property name="text">
         <string>GetWrapper</string>
        </property>
       </widget>
      </item>
      <item row="8" column="1">
       <widget class="QCheckBox" name="transferThisCb">
        <property name="text">
         <string>TransferThis</string>
        </property>
       </widget>
      </item>
      <item row="9" column="0">
       <widget class="QCheckBox" name="inCb">
        <property name="text">
         <string>In</string>
        </property>
       </widget>
      </item>
      <item row="9" column="1">
       <layout class="QFormLayout" name="formLayout_3">
        <item row="0" column="0">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>TypeHint</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="typeHint"/>
        </item>
       </layout>
      </item>
      <item row="10" column="0">
       <widget class="QCheckBox" name="keepRefCb">
        <property name="text">
         <string>KeepReference</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>16</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
  <zorder>buttonBox</zorder>
  <zorder>groupBox</zorder>
  <zorder>verticalSpacer</zorder>
 </widget>
 <tabstops>
  <tabstop>name</tabstop>
  <tabstop>unnamedCb</tabstop>
  <tabstop>pyType</tabstop>
  <tabstop>allowNoneCb</tabstop>
  <tabstop>arrayCb</tabstop>
  <tabstop>arraySizeCb</tabstop>
  <tabstop>constrainedCb</tabstop>
  <tabstop>disallowNoneCb</tabstop>
  <tabstop>docType</tabstop>
  <tabstop>docValue</tabstop>
  <tabstop>encodingCb</tabstop>
  <tabstop>getWrapperCb</tabstop>
  <tabstop>inCb</tabstop>
  <tabstop>keepRefCb</tabstop>
  <tabstop>noCopyCb</tabstop>
  <tabstop>outCb</tabstop>
  <tabstop>pyIntCb</tabstop>
  <tabstop>ref</tabstop>
  <tabstop>resultSizeCb</tabstop>
  <tabstop>singleShotCb</tabstop>
  <tabstop>transferCb</tabstop>
  <tabstop>transferBackCb</tabstop>
  <tabstop>transferThisCb</tabstop>
  <tabstop>typeHint</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ArgPropertiesBase</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ArgPropertiesBase</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
